src/Skia/Perspex.Skia/SkiaPlatform.cs

```diff
@@ -30,7 +30,10 @@ public static bool ForceSoftwareRendering
             set
             {
                 s_forceSoftwareRendering = value;
-                MethodTable.Instance.SetOption(MethodTable.Option.ForceSoftware, new IntPtr(value ? 1 : 0));
+
+                // Do we still need this with SkiaSharp??
```

Comment:
Is it possible to answer questions like this before merging to master? Because I sure as hell don't know the answer ;)